### ISSUE TITLE
Backup

### ISSUE THREAD

**Issue by johnp1954** [syscoin/blockmarket-desktop-public#361]
1) Please in the very next release is it possible to have a 'Backup Wallet' option probably accessed from the same place as Import/Dump Wallet.
2) Is it also possible when pressing New Alias to default to 'Basic'

**Comment by benp1000** [comment]
agreed important to reduce help issues and create better practices with new users/less technically proficient to have a more intuitive backup option

**Comment by jnessbcf** [comment]
Hi!

“Backup Wallet” - This is a great suggestion and we’ll slot it into one of the next couple of releases following “the very next release”, as the features and functions for the very next release have already been determined.  :)(See Slack regarding “post BMD3 release” expectations).

As for the alias piece stay tuned.
